src/main/kotlin/com/team4099/robot2021/subsystems/Vision.kt

```diff
@@ -0,0 +1,84 @@
+package com.team4099.robot2021.subsystems
+
+import com.team4099.lib.logging.Logger
+import com.team4099.lib.units.base.Length
+import com.team4099.lib.units.base.inInches
+import com.team4099.lib.units.base.inMeters
+import com.team4099.lib.units.derived.degrees
+import com.team4099.lib.units.derived.inRadians
+import com.team4099.lib.units.derived.tan
+import com.team4099.robot2021.config.Constants
+import edu.wpi.first.wpilibj2.command.SubsystemBase
+import org.photonvision.PhotonCamera
+
+object Vision : SubsystemBase() {
+  //for up close
+  private val camera : PhotonCamera = PhotonCamera("gloworm")
+  //from farther away
+  private val zoomedCamera : PhotonCamera = PhotonCamera("")
```

Comment:
we never use the second gloworm in the code, we need to do that bc at some distances the other can't see the target